### PR TITLE
Update to 2.4.57: bugs and improvements

### PR DESCRIPTION
**Pull request by reporter4u**
Hi,

a new Httpd version has been released with some bugs fix and improvements.

I hope you can validate it and do reviews.

Thank you.

Roberto

### CODE REVIEW

File: httpd24u.spec [hunk 1]
```diff
@@ -827,6 +827,9 @@ exit $rv
 
 
 %changelog
+* Thu Mar 09 2023 Reporter4u <reporter4u@gmail.com> - 2.4.57-1
```
Comment (jeffsheltren) [inline httpd24u.spec:830]: Can you correct this date please?

**Review by jeffsheltren — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks like the date used in the latest changelog is incorrect though.

File: httpd24u.spec (hunk 1 above)
Comment (reporter4u) [reply httpd24u.spec]: I did it.

Thank you.

Roberto

**Review by jeffsheltren — APPROVED** [review]

**Comment by jeffsheltren** [comment]
httpd24u-2.4.57-1.el7.ius has been pushed to IUS Testing

**Pull request merged**